pkg/kubectl/genericclioptions/record_flags.go

```diff
@@ -150,3 +155,36 @@ func (r *ChangeCauseRecorder) MakeRecordMergePatch(obj runtime.Object) ([]byte,
 
 	return jsonpatch.CreateMergePatch(oldData, newData)
 }
+
+// stringifyCommand will stringify and return all environment arguments ie. a command run by a client
+// using the factory.
+// Set showSecrets false to filter out stuff like secrets.
+func stringifyCommand(cmd *cobra.Command) string {
```

Comment:
We need a better name, please. `stringifyCommand` completely hides the functionality of this command.

Reply:
`classifyCommandOutput`, `hideCommandSecrets` or something like that.

Reply:
And I agree with David, move this to one of the general helpers and make it public.

Reply:
Ignore what I said above, I thought about something else ;)